src/runtime/meter-toast-api.js

```diff
@@ -327,11 +330,13 @@ export class MeterToastApi {
    * @private
    */
   startNativeFlow_(response) {
+    this.removeCloseEventListener();
+    // We shouldn't decrement the meter on redirects, so don't call onConsumeCallback.
+    this.onConsumeCallbackHandled_ = true;
     if (response.getNative()) {
-      this.removeCloseEventListener();
-      // We shouldn't decrement the meter on redirects, so don't call onConsumeCallback.
-      this.onConsumeCallbackHandled_ = true;
       this.deps_.callbacks().triggerSubscribeRequest();
+    } else {
+      this.deps_.callbacks().triggerOffersFlowRequest();
```

Comment:
I wouldn't expect startNativeFlow_ to handle non-native.

Reply:
Rename it to `startSubscriptionFlow_`